docs/io.cozy.calendar.md

```diff
@@ -0,0 +1,54 @@
+[Table of contents](README.md#table-of-contents)
+    
+# Cozy Calendar doctype
+
+## `io.cozy.calendar`
+
+The `io.cozy.calendar` doctype is loosely based on the [iCalendar RFC](https://datatracker.ietf.org/doc/html/rfc5545), in that some of the attributes have been renamed for clarity. The attributes with a `?` are optional.
+
+- `start?` : {date}
+- `end?` : {date}
+- `label?` : {string}
+- `description?` : {string}
+- `location?` : {string}
+- `organizer?` : {string}
+- `status?` : {string} `CONFIRMED|CANCELLED`
+- `attendee?` : {array} Array of string with event attendees
```

Comment:
I think we need to add the timezone, typically with a tzid attribute: https://icalendar.org/iCalendar-RFC-5545/3-2-19-time-zone-identifier.html
This way, we can correctly deal with winter/summer time changes.

Reply:
Likewise, we might add a recurrence attribute, i.e. rrule: https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html

Reply:
And final request, @ecnivtwelve would you mind adding a link to each iCal corresponding attribute? :pray: 
Like the ones I gave above for timezone and recurrence rule

Reply:
Added here -> https://github.com/cozy/cozy-doctypes/commit/d4ee994331fbd3133447dfa8e7744ce3766d441e